src/styles/_typography.scss

```diff
@@ -1,3 +1,19 @@
 h1 {
   @extend %h1;
 }
```

Comment:
In the file `_typography.scss` are usually written common font-styles.
Better to move everything from here to extends, and write styles for your `headings` and `p` in `main.scss`